README.md

```diff
@@ -123,6 +123,26 @@ let x = 0.0
 end
 ```
 
+Note that the let block is used here to create a scope to allow assignment of 
```

Comment:
Not really related, but I don't get why `let` is presented here. AFAICT it isn't specific to `@byrow!` and isn't particularly useful with that macro (at least not more than in other situations).